src/Symfony/Bundle/SecurityBundle/DependencyInjection/SecurityExtension.php

```diff
@@ -191,6 +191,10 @@ private function createAuthorization(array $config, ContainerBuilder $container)
                 $attributes[] = $this->createExpression($container, $access['allow_if']);
             }
 
+            if (!$attributes) {
```

Comment:
What about instead checking that at least one of the keys in `$access` is not `null`?

Reply:
or the empty array in case of `method`, `ips` and `roles`